Code/EnginePlugins/VisualScriptPlugin/Runtime/VisualScript_inl.h

```diff
@@ -87,6 +87,24 @@ EZ_ALWAYS_INLINE ezVisualScriptGraphDescription::DataOffset ezVisualScriptGraphD
   return {};
 }
 
+EZ_ALWAYS_INLINE ezVisualScriptGraphDescription::DataOffset* ezVisualScriptGraphDescription::Node::GetInputDataOffsets()
+{
+  return m_NumInputDataOffsets <= EZ_ARRAY_SIZE(m_InputDataOffsets.m_Embedded) ? m_InputDataOffsets.m_Embedded : m_InputDataOffsets.m_Ptr;
+}
+
+EZ_ALWAYS_INLINE ezVisualScriptGraphDescription::DataOffset* ezVisualScriptGraphDescription::Node::GetOutputDataOffsets()
+{
+  return m_NumOutputDataOffsets <= EZ_ARRAY_SIZE(m_OutputDataOffsets.m_Embedded) ? m_OutputDataOffsets.m_Embedded : m_OutputDataOffsets.m_Ptr;
+}
+
+// static
+template <typename T>
+EZ_ALWAYS_INLINE constexpr ezUInt32 ezVisualScriptGraphDescription::Node::GetUserDataAlignment()
+{
+  // Ensures at least 8 byte alignment, thus making it compatible between 32 and 64 bit platforms.
+  return ezMath::Max<ezUInt32>(EZ_ALIGNMENT_OF(T), 8u);
```

Comment:
I never understood why EZ_ALIGNMENT_OF works the way it does. Is this really the correct choice here? Because IIRC it also uses some MAX value internally.

Reply:
Looks like we don't need the EZ_ALIGNMENT_OF macro anymore, at least it seems to work in msvc. Let's see what CI and the other platforms say.